semgrep/tests/e2e/snapshots/test_check/test_timeout/results.json

```diff
@@ -3,7 +3,7 @@
     {
       "code": 2,
       "level": "warn",
-      "message": "Semgrep Core WARN - Timeout: When running rules.forcetimeout on targets/equivalence/open_redirect.py: ",
+      "message": "Semgrep Core WARN - Timeout: When running rules.forcetimeout on targets/equivalence/open_redirect.py: Main.timeout_function:1",
```

Comment:
Hmm I don't think this is right?

Reply:
It's the actual message that's associated with the exception in `semgrep-core`---I can override it if you think it's too confusing for users

Reply:
I think we can keep it if it's useful for us to debug but I imagine it's always going to timeout on this main function? If so overriding would be great :D